index.html

```diff
@@ -203,21 +309,34 @@ <h2>About me</h2>
 
     <section id="contact">
       <form action="https://formspree.io/f/mqkonqbq" method="POST">
+        <h5>I'm always <br>interested in hearing about new projects, so if you'd like to chat please get in touch.</h5>
+
         <ul id="form-ul">
-          <li>I'm always <br>interested in hearing about new projects, so if you'd like to chat please get in touch.</li>
-          <li>
+          <li id="li-full-name">
             <input placeholder="Fullname" name="fullname" 
             type="text" maxlength="30" required>
           </li>
-          <li>
+
+          <li id="li-first-name">
+            <input
+            placeholder="First-name" name="first-name" 
+            type="text" maxlength="30" required>
+          </li>
+
+          <li id="li-last-name">
+            <input  placeholder="Last-name" name="last-name" 
+            type="text" maxlength="30" required>
+          </li>
+
+          <li id="li-email">
```

Comment:
- [ ] As suggested by the previous reviewer please make sure your message `textarea` is visible to the users. In the figma design there is a textarea with no borders but you can still clearly see a `textarea` because of the `background-color` of the field which is different from `background-color` of the contact form. I would also suggest you to use `#FBF8F7` for the `background-color` of your textarea field to make it easily visible. 

| Figma design | Your design |
|--------------|--------------|
|<img width="794" alt="Screenshot 2023-05-05 at 10 14 39 AM" src="https://user-images.githubusercontent.com/44798044/236382023-69793d44-c74c-48ba-85a6-639b65b1fcd5.png">|<img width="565" alt="Screenshot 2023-05-05 at 10 12 34 AM" src="https://user-images.githubusercontent.com/44798044/236381831-358d30d1-9238-455a-a58e-b18902ee0972.png">|

Reply:
Yeah, not easy to see this with normal screen.
Thank you Shella